src/tty/unix.rs

```diff
@@ -58,6 +61,11 @@ fn make_pty(size: winsize) -> (RawFd, RawFd) {
     win_size.ws_xpixel = 0;
     win_size.ws_ypixel = 0;
 
+    unsafe {
+        OLD_ROW = win_size.ws_row;
+        OLD_COL = win_size.ws_col;
+    };
```

Comment:
In Rust, things are usually marked as `unsafe` for a reason and adding unsafe code when there is an alternative is strictly discouraged.

In a multi-threaded context, this could easily cause problem which is why variables should be stored on structs instead of the global scope.

Reply:
Thanks for your feedback.

I've adapted everything to your proposals.